jetty/src/main/scala/com/avsystem/commons/jetty/rpc/JettyRPCFramework.scala

```diff
@@ -1,21 +1,20 @@
 package com.avsystem.commons
 package jetty.rpc
 
-import java.nio.charset.StandardCharsets
 import com.avsystem.commons.rpc.StandardRPCFramework
 import com.avsystem.commons.serialization.json.{JsonStringInput, JsonStringOutput, RawJson}
 import com.avsystem.commons.serialization.{GenCodec, HasGenCodec}
 import com.typesafe.scalalogging.LazyLogging
-
-import javax.servlet.http.{HttpServletRequest, HttpServletResponse}
-import org.eclipse.jetty.client.HttpClient
-import org.eclipse.jetty.client.api.Result
-import org.eclipse.jetty.client.util.{BufferingResponseListener, StringContentProvider, StringRequestContent}
+import jakarta.servlet.http.{HttpServlet, HttpServletRequest, HttpServletResponse}
+import org.eclipse.jetty.client.{BufferingResponseListener, HttpClient, Result, StringRequestContent}
+import org.eclipse.jetty.ee10.servlet.ServletContextHandler
 import org.eclipse.jetty.http.{HttpMethod, HttpStatus, MimeTypes}
-import org.eclipse.jetty.server.handler.AbstractHandler
-import org.eclipse.jetty.server.{Handler, Request}
+import org.eclipse.jetty.server.Handler
 
-import scala.concurrent.duration._
+import java.nio.charset.StandardCharsets
+import java.util.concurrent.atomic.AtomicBoolean
+import scala.concurrent.duration.*
```

Comment:
Nit: switch to Scala 3 syntax? IMO we should do it in all classes or keep using Scala 2 _